README.md

```diff
@@ -1,18 +0,0 @@
-# CoMPAS Architecture
```

Comment:
Removing the current readme if fine. It would be nice to have a readme.me for github visitors to see what this repository is all about.  We can fix this later. @Flurb can you file an issue for this?

Reply:
Good idea! I'll make it!

Reply:
https://github.com/com-pas/compas-architecture/issues/14